src/ripple/server/impl/Port.cpp

```diff
@@ -205,6 +205,29 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
         }
     }
 
+    {
+        auto const result = section.find("send_queue_limit");
+        if (result.second)
+        {
+            try
+            {
+                port.ws_queue_limit =
+                    beast::lexicalCastThrow<std::uint16_t>(result.first);
+
+                // Queue must be greater than 0
```

Comment:
I'm not fond of such "magic" values and, anyways, who _really_ wants an _unlimited_ queue size? I'd rather expand the field to `std::uint32_t`. This allows people to set their queue size to numbers that, effectively, emulate an unlimited queue, without the need for the special case.